wordpress/README.md

```diff
@@ -59,6 +59,8 @@ curl -SL http://wordpress.org/latest.tar.gz | tar --strip 1 -xzf -
 
 ````
 
+Use the Caddyfile in this example and make sure that fastcgi is listening on port 9000
+
```

Comment:
Can you also update the last point in the Troubleshooting section below? Now that the example Caddyfile shows a TCP port instead of a UNIX socket, we can invert the recommendation: try a Unix socket if they have trouble with a TCP port...